tool_collections/cufflinks/cuff_macros.xml

```diff
@@ -4,6 +4,7 @@
   <xml name="requirements">
     <requirements>
       <requirement type="package" version="2.2.1">cufflinks</requirement>
+      <requirement type="package" version="2.7.15">python</requirement>
```

Comment:
If we add a requirement here we need to bump the version of all tools in `tool_collections/cufflinks` , but some of these don't actually use a Python script any more. So this should be moved to the cufflinks tool XML (and maybe use a more recent Python version works?).
Both cuffcompare and cuffmerge wrappers presently use a Python script, but that could be replaced with Cheetah code.

Also, it may make sense to move this tool collection to IUC, I think?

Reply:
I transferred cufflinks to IUC. Py 3.6 seems to work (3.7 gives conflicts).

Reply:
@bernt-matthias The current practice for tools moved to IUC is removing them from this repo, not moving them to `deprecated/` dir.